tests/test_verifylib.py

```diff
@@ -395,26 +395,45 @@ def setUp(self):
     self.materials = {
       "foo": {"sha256": self.sha256_foo},
       "foobar": {"sha256": self.sha256_foobar},
+      "sub/foobar": {"sha256": self.sha256_foobar},
       "sub/foo": {"sha256": self.sha256_foo},
-      "sub/foobar": {"sha256": self.sha256_foobar}
-
+      "lib/build/foobar": {"sha256": self.sha256_foobar},
+      "lib/foo": {"sha256": self.sha256_foo},
+      "lib/build/foo": {"sha256": self.sha256_foo},
+      "lib/foobar": {"sha256": self.sha256_foobar},
+      "build/lib": {"sha256": self.sha256_foobar},
```

Comment:
Here as well, let's reserve `lib` etc for directory names to avoid confusion down the line. Perhaps this can be `build/foobar` since the hash is that of `foobar` anyway.